editor/plugins/script_editor_plugin.cpp

```diff
@@ -2798,6 +2798,11 @@ void ScriptEditor::_reload_scripts(bool p_refresh_only) {
 				scr->set_source_code(rel_scr->get_source_code());
 				scr->set_last_modified_time(rel_scr->get_last_modified_time());
 				scr->reload(true);
+
+				for (const DocData::ClassDoc &cd : scr->get_documentation()) {
+					EditorHelp::get_doc_data()->add_doc(cd);
+					update_doc(cd.name);
+				}
```

Comment:
This is the only actual change to this PR. The rest is just copying docs to `EditorHelp` more efficient.